src/viz-population-over-time/VizPopulationOverTime.js

```diff
@@ -47,148 +36,159 @@ const ChartWrapper = styled(BaseChartWrapper)`
-              stroke: THEME.colors.background,
-              strokeWidth: 1,
-            })}
-            lineType={{ type: "stackedarea", sort: null }}
-            margin={MARGIN}
-            pointStyle={{ display: "none" }}
-            responsiveWidth
-            // showLinePoints="top"
-            size={[undefined, 430]}
-            xAccessor="time"
-            xScaleType={scaleTime()}
-            yAccessor="population"
-            yExtent={[0]}
-          />
-        </ResponsiveTooltipController>
-      </ChartWrapper>
-      <LegendWrapper>
-        <ColorLegend
-          highlighted={highlighted}
-          items={chartData}
-          setHighlighted={setHighlighted}
-        />
-      </LegendWrapper>
-    </Wrapper>
+    <Measure bounds>
+      {({
+        measureRef,
+        contentRect: {
+          bounds: { width },
+        },
+      }) => (
+        <Wrapper ref={measureRef}>
+          <ChartWrapper>
+            <ResponsiveTooltipController
+              getTooltipProps={(d) => {
+                const dateHovered = d.time;
+                return {
+                  title: getDateLabel(dateHovered),
+                  records: getDataWithPct(
+                    d.points
+                      .filter((p) => isEqual(p.data.time, dateHovered))
+                      .map((p) => ({
+                        label: p.parentLine.label,
+                        value: p.data.population,
+                      }))
+                  ),
+                };
+              }}
+              hoverAnnotation={[
+                { type: "x", disable: ["connector", "note"] },
+                { type: "frame-hover" },
+              ]}
+            >
+              <MinimapXYFrame
+                axes={[
+                  {
+                    orient: "left",
+                    tickFormat: formatAsNumber,
+                    tickLineGenerator: () => null,
+                  },
+                  {
+                    orient: "bottom",
+                    tickFormat: (time) =>
+                      time.getDate() === 1 ? format(time, "MMM y") : null,
+                    tickLineGenerator: () => null,
+                    ticks: isMobile ? 5 : 10,
+                  },
+                ]}
+                baseMarkProps={BASE_MARK_PROPS}
+                lines={data}
+                lineStyle={(d) => ({
+                  fill:
+                    highlighted && highlighted.label !== d.label
+                      ? highlightFade(d.color)
+                      : d.color,
+                  stroke: THEME.colors.background,
+                  strokeWidth: 1,
+                })}
+                lineType={{ type: "stackedarea", sort: null }}
+                margin={MARGIN}
+                matte
+                minimap={{
+                  axes: [],
+                  baseMarkProps: BASE_MARK_PROPS,
+                  brushEnd: (brushExtent) => {
+                    const [start, end] = brushExtent || [];
```

Comment:
Do we need to do any validation about which is start and which is end, i.e. that they're not out of order, or is that taken care of under the hood?

Reply:
it's not necessary, the order doesn't matter. you can drag the start past the end or vice versa and everything will still display correctly